cmake/dependencies.cmake

```
@@ -84,8 +84,10 @@ if(TUVX_ENABLE_TESTS)
       CACHE BOOL "" FORCE)
 
   FetchContent_MakeAvailable(googletest)
-
   # don't run clang-tidy on google test
   set_target_properties(gtest PROPERTIES CXX_CLANG_TIDY "")
   set_target_properties(gtest_main PROPERTIES CXX_CLANG_TIDY "")
 endif()
+
+# ##############################################################################
+# ##############################################################################
```

```suggestion
################################################################################
################################################################################
```
or remove if it's at the end of the file